src/actions/articles/index.js

```diff
@@ -24,5 +25,6 @@ export {
   getArticleHighlights,
   clearGetArticleHighlightsStore,
   deleteArticleHighlight,
-  clearDeleteArticleHighlightStore
+  clearDeleteArticleHighlightStore,
+  bookmarkArticle
```

Comment:
I think this should go in the index of bookmarks, not in articles

Reply:
ok let me work on it